spec/components/calendar_spec.rb

```diff
@@ -0,0 +1,25 @@
+describe "CalendarComponent", type: :view do
```

Comment:
Another boring comment, but I think we should use `RSpec.describe` consistently. Just using `describe` is valid too, but I think most of our test suites use the `RSpec` namespace.